delta/src/main/scala/io/qbeast/spark/delta/DeltaMetadataWriter.scala

```diff
@@ -176,7 +177,7 @@ private[delta] case class DeltaMetadataWriter(
       registerStatsTrackers(statsTrackers)
 
       // Execute write
-      val (tableChanges, indexFiles, deleteFiles) = writer
+      val (tableChanges, indexFiles, deleteFiles) = writer(txn.txnStartTimeNs.toString)
```

Comment:
I need a bit more of understanding here. Why the commitTime is the start transaction time?

Reply:
And isn't it better if we manipulate it as Long?

Reply:
Nevertheless, instead of `commitTime`, we should use a more verbose naming (such as `startCommitTime`) to avoid confusion in the future.
 Also, do you think it should be saved somewhere? I know it's Hudi-specific, but should we include it in Qbeast protocol too?

Reply:
Or `txnStartTime,` as it is created when the transaction starts?

Reply:
Even before any parquet file is written

Reply:
I would suggest using something more general than `txn` ... although the three formats are transaction-based, the abbreviation is widely used in Delta Lake only I believe.

Maybe `transactionStartTime` if you all agree 😆 😝 🙄  `commitStartTime` feels ok too

Github should allow polls for this.........

Reply:
In hudi they call it `instantTime`, so i proceed with `commitStartTime` 😆

Reply:
IMHO, `transactionStartTime` does a better job of conveying the nature of this parameter.

The thing is that the time is not exactly the commit start time.
It is created when we start indexing and analyzing the data.
Then, it is written, which takes a huge chunk of time, and then, finally, it starts with the commit.

Reply:
That is a good point actually..

Reply:
Changed